### PR TITLE
Aleksei/height fix

### PR DESCRIPTION
**Pull request by iambeone**
Closes #ISSUE

**Description:**

Before PR 
image

After PR
image

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: src/components/common/TmPage.vue [hunk 1]
```diff
@@ -133,7 +133,7 @@ export default {
 
 .tm-page-main {
   position: relative;
-  min-height: 100vh;
+  /*min-height: 100vh;*/
```
Comment (Bitcoinera) [inline src/components/common/TmPage.vue]: @jbibla should we just remove this? It seems using the `vh` was causing some undesired side-effects for bigger screens

**Comment by jbibla** [comment]
the 100vh is for this case:

Screen Shot 2020-03-20 at 9 10 26 AM

**Comment by iambeone** [comment]
But it's shouldn't be 100vh. We can use flex here))

**Comment by jbibla** [comment]
flex being used!

**Pull request merged**